app/controllers/outgoings_controller.rb

```diff
@@ -0,0 +1,38 @@
+class OutgoingsController < ApplicationController
+  # api! works but the links in resulting docs are broken. See https://github.com/Apipie/apipie-rails/issues/559
+  api :POST, 'assessments/:assessment_id/outgoings', 'Create outgoings'
+  formats ['json']
+  param :assessment_id, :uuid, required: true
+  param :outgoings, Array, desc: 'Collection of outgoings', required: true do
+    param :type_of_outgoing, Outgoing.type_of_outgoings.values, required: true, desc: 'The type of product or service the outgoing paid for'
+    param :payment_date, Date, date_option: :today_or_older, required: true, desc: 'The date the outgoing was paid'
```

Comment:
i wonder if we should use `:date` instead of `Date` for consistency because we use `:uuid` and `:currency`

Reply:
`Date` matches the usage in other places. For example line 6 `param :outgoings, Array, ....`

If you look at the apipie documentation, it appears that the default is to pass in the object class if you can, and then use a :symbol description when that's not possible.

Reply:
Alright that makes sense